### DISCUSSION TITLE
E2 and O1 Interface Support in Current srsRAN Release

### DISCUSSION THREAD

**Discussion by Moussa-Guemdani** [srsran/srsRAN_Project#901] [General Help] [answered]
Hello srsRAN team,

I have a couple of questions regarding the current release of the srsRAN project:

- Does the latest release of srsRAN include E2 interface support for the O-DU?

- Does the current release provide O1 interface support for the O-CU, O-DU, and O_RU components as well as A1 support for Near-Real Time RIC?

Thank you for your help!

**Comment by pgawlowicz** [comment]
Hi,
The current project release supports an E2 agent for DU and DU.
A1 interface is between Real-Time and Non-Real-Time RICs, so it is not a part of the srsRAN-Project. 
O1 will be only available in the Enterprise version.
Best,
Piotr

**Reply by Moussa-Guemdani** [reply]
Hello,
Does srsRAN Near-RT RIC support external Non-RT RICs?
E.g: a developer wants to deploy an external Non-RT RIC to provide policies to srsRAN Near-RT RIC.

**Reply by pgawlowicz** [reply, marked as answer]
Hi, we only expose E2 interface to connect a Near-RT RIC. There is no srsRAN Near-RT RIC.
